index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
+
+  <link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
+  <link href="https://fonts.googleapis.com/css2?family=Big+Shoulders+Display&family=Lexend+Deca&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  
+  <title>Frontend Mentor | 3-column preview card component</title>
+
+  <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
+  <style>
+    .attribution { font-size: 11px; text-align: center; margin-top: 20px; }
+    .attribution a { color: hsl(228, 45%, 44%); }
+  </style>
+</head>
+<body>
+  
+  <div class="container">
+    <div class="cards">
+      <div class="card-one">
+        <img src="images/icon-sedans.svg" alt="sedans icon">
+        <div class="card-title">
+          <h1>Sedans</h1>
+        </div>
+        <div class="card-description">
+          <p>Choose a sedan for its affordability and excellent fuel economy. Ideal for cruising in the city 
+            or on your next road trip.</p>
+          <button><a href="/sedans"></a>Learn More</button>
```

Comment:
For accessibility concerns, it is not considered good practice to put an interactive element inside another one. From the design, it looks like a link with styling. So you can make the link look like that :)

Reply:
So I should remove the button tags??